### PR TITLE
Automatically porting System.Text.Unicode triple slash comments

### PR DESCRIPTION
**Pull request by carlossanlop**
Found an additional batch of comments that could be ported.

### CODE REVIEW

**Comment by carlossanlop** [comment]
For language review @mairaw @rpetrusha 

Adding area owners of **System.Text.Encoding**:  @layomia, @krwq, @tarekgh

Question for area owners: Are you also the owners of System.Text.Unicode? If not, should we add a new entry to [issue-guide.md](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/issue-guide.md)? If yes, should we add a note next to your mentions so that people know that the **System.Text.Unicode** owners are the same as **System.Text.Encoding**?

**Comment by tarekgh** [comment]
@carlossanlop System.Text.Encoding is part of "System.Text.Encodings.Web" area. @GrabYourPitchforks is the owner. you can assume me as the backup owner.

**Comment by carlossanlop** [comment]
@tarekgh thanks for the clarification. I'll add the note to the issue-guide.md file.

**Comment by carlossanlop** [comment]
@tarekgh wait, both System.Text.Encoding and System.Text.Encoding.Web are already in the issue-guide table. The one that is not clearly indicated is System.Text.Unicode. Were you referring to that one?

**Comment by tarekgh** [comment]
@carlossanlop we don't have area-label for System.Text.Unicode. it is simplicity part of System.Text.Encoding.Web. Note, we don't create area tag for every namespace except if we expect this will have many issues need to be tracked by this specific namespace.

Anyway, I added entry to the issue guide making this namespace points to System.Text.Encoding.Web.

File: xml/System.Text.Unicode/UnicodeRanges.xml [hunk 1]
```diff
@@ -77,7 +77,11 @@ Currently, the <xref:System.Text.Unicode.UnicodeRange> class supports only named
       <Docs>
         <summary>Gets the Alphabetic Presentation Forms Unicode block (U+FB00-U+FB4F).</summary>
         <value>The Alphabetic Presentation Forms Unicode block (U+FB00-U+FB4F).</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+See http://www.unicode.org/charts/PDF/UFB00.pdf for the full set of characters in this block.
```
Comment (mairaw) [inline xml/System.Text.Unicode/UnicodeRanges.xml]:
```suggestion

## Remarks

For the full set of characters in this block, see [The Unicode Standard, Version 12.1
 - Alphabetic Presentation Forms](https://www.unicode.org/charts/PDF/UFB00.pdf).

```

File: xml/System.Text.Unicode/UnicodeRanges.xml [hunk 2]
```diff
@@ -77,7 +77,11 @@ Currently, the <xref:System.Text.Unicode.UnicodeRange> class supports only named
...
+          ]]></format>
```
Comment (mairaw) [inline xml/System.Text.Unicode/UnicodeRanges.xml]:
```suggestion
 ]]></format>
```

File: xml/System.Text.Unicode/UnicodeRanges.xml [hunk 3]
```diff
@@ -102,7 +106,11 @@ Currently, the <xref:System.Text.Unicode.UnicodeRange> class supports only named
       <Docs>
         <summary>Gets the Arabic Unicode block (U+0600-U+06FF).</summary>
         <value>The Arabic Unicode block (U+0600-U+06FF).</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+See http://www.unicode.org/charts/PDF/U0600.pdf for the full set of characters in this block.
```
Comment (mairaw) [inline xml/System.Text.Unicode/UnicodeRanges.xml]:
```suggestion

## Remarks

For the full set of characters in this block, see [The Unicode Standard, Version 12.1 - Arabic](https://www.unicode.org/charts/PDF/U0600.pdf).

```

File: xml/System.Text.Unicode/UnicodeRanges.xml [hunk 4]
```diff
@@ -102,7 +106,11 @@ Currently, the <xref:System.Text.Unicode.UnicodeRange> class supports only named
...
+          ]]></format>
```
Comment (mairaw) [inline xml/System.Text.Unicode/UnicodeRanges.xml]:
```suggestion
 ]]></format>
```

File: xml/System.Text.Unicode/UnicodeRanges.xml [hunk 5]
```diff
@@ -127,7 +135,11 @@ Currently, the <xref:System.Text.Unicode.UnicodeRange> class supports only named
       <Docs>
         <summary>Gets the Arabic Extended-A Unicode block (U+08A0-U+08FF).</summary>
         <value>The Arabic Extended-A Unicode block (U+08A0-U+08FF).</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+See http://www.unicode.org/charts/PDF/U08A0.pdf for the full set of characters in this block.
```
Comment (mairaw) [inline xml/System.Text.Unicode/UnicodeRanges.xml]:
```suggestion

## Remarks

For the full set of characters in this block, see [The Unicode Standard, Version 12.1 - Arabic Extended-A](https://www.unicode.org/charts/PDF/U08A0.pdf).

```

File: xml/System.Text.Unicode/UnicodeRanges.xml [hunk 6]
```diff
@@ -127,7 +135,11 @@ Currently, the <xref:System.Text.Unicode.UnicodeRange> class supports only named
...
+          ]]></format>
```
Comment (mairaw) [inline xml/System.Text.Unicode/UnicodeRanges.xml]:
```suggestion
 ]]></format>
```

**Comment by carlossanlop** [comment]
@mairaw this one's fully baked. Can we get it merged?

**Comment by mairaw** [comment]
We'll do one more review pass now and merge if it looks good! Thanks for the ping.

**Pull request merged**